packages/app/src/components/pages/Home/NewFilesToReview.js

```diff
@@ -3,20 +3,43 @@ import { NominationsDataContext } from '../../../utils/context/NominationsContex
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import NewNomination from './NewNomination';
 import styles from "./styles.css";
+import useSort from './useSort'
 
 const NewFilesToReview = () => {
   const [NominationsData, setNominationsData] = useContext(NominationsDataContext);
   const [showAll, setShowAll] = useState(false);
   const receivedNominations = NominationsData ? NominationsData.filter(nominations => nominations.status === "received") : []
+  const { sortedNoms, requestSort, sortConfig } = useSort()
+  const sortedNominations = sortedNoms ? sortedNoms.filter(nominations => nominations.status === "received") : []
 
   const handleClick = () => {
     setShowAll(!showAll)
   }
 
   const conditionalNominationRender = () => {
-    return showAll ? receivedNominations : receivedNominations.slice(0,3)
+    if (sortedNoms && showAll) {
+      return sortedNominations
+    }
+    if (sortedNoms && !showAll) {
+      return sortedNominations.slice(0,3)
+    }
+    if (!sortedNoms && showAll) {
+      return receivedNominations
+    }
+    if (!sortedNoms && !showAll) {
+      return receivedNominations.slice(0,3)
+    }
```

Comment:
I'm still confused why we need both `sortedNominations` and `receivedNominations`.